packages/flutter_tools/lib/src/project.dart

```diff
@@ -8,22 +8,23 @@ import 'package:yaml/yaml.dart';
 
 import '../src/convert.dart';
 import 'android/gradle_utils.dart' as gradle;
-import 'artifacts.dart';
 import 'base/common.dart';
+import 'base/error_handling_io.dart';
 import 'base/file_system.dart';
 import 'base/logger.dart';
-import 'build_info.dart';
+import 'base/utils.dart';
 import 'bundle.dart' as bundle;
-import 'cmake.dart';
+import 'cmake_project.dart';
 import 'features.dart';
 import 'flutter_manifest.dart';
 import 'flutter_plugins.dart';
 import 'globals_null_migrated.dart' as globals;
-import 'ios/plist_parser.dart';
-import 'ios/xcode_build_settings.dart' as xcode;
-import 'ios/xcodeproj.dart';
 import 'platform_plugins.dart';
 import 'template.dart';
+import 'xcode_project.dart';
+
+export 'cmake_project.dart';
+export 'xcode_project.dart';
```

Comment:
Also happy to instead not export these and instead update all imports, including in g3.